iXBRLViewerPlugin/viewer/src/js/fact.js

```diff
@@ -77,11 +77,11 @@ export class Fact {
             else {
                 formattedNumber = formatNumber(v, d);
             }
-            if (this.isMonetaryValue()) {
-                v = this.measureLabel() + " " + formattedNumber;
+            if (this.isMonetaryValue() && this.unit().isSimple()) {
```

Comment:
See separate comment - I think "simple" should be part of the definition of monetary.